content/get-haiku/installation-guide.html

```diff
@@ -225,24 +225,24 @@ <h3>Not using GRUB</h3>
 
 <h3>Boot using the CD</h3>
 
-<p>You can also use the boot CD to boot into your Haiku installation without adding it to a boot manager. To do so, hold <span class="key">SHIFT</span> before the CD begins to load. This will bring up the Boot Loader Options.</p>
+<p>You can also use the boot CD to boot into your Haiku installation without adding it to a boot manager. To do so, hold <span class="button">SHIFT</span> before the CD begins to load. This will bring up the Boot Loader Options.</p>
 
 <p><figure role="group">
 	<a id="step-2-1" name="step-2-1"></a>
 	<img src="/files/images/installation_guide/19-bootloader-menu.png" alt="The Boot Loader Menu" />
-<figcaption><p class="step">1</p><p class="caption">To choose the installation to boot from, press <span class="key">ENTER</span> to navigate to the boot volume selection. Your installation should be listed there.</p>
+<figcaption><p class="step">1</p><p class="caption">To choose the installation to boot from, press <span class="button">ENTER</span> to navigate to the boot volume selection. Your installation should be listed there.</p>
```

Comment:
Why the change from 'key' to 'button'? Those are all keys...

Reply:
@humdingerb The first part of the documentation used the "button" span, so I just used the "button" span for the whole thing.

Reply:
Actually, never mind - I just realised my mistake - I will try and fix it 😆